src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/LocationScheduleUsage.kt

```diff
@@ -3,6 +3,5 @@ package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model
 enum class LocationScheduleUsage {
   COURT,
   PROBATION,
-  SHARED,
   BLOCKED,
```

Comment:
Two enums with same name, same values?

Reply:
One is model based the other is entity based.